src/uk/gov/defra/ffc/ADO.groovy

```diff
@@ -1,25 +1,41 @@
 package uk.gov.defra.ffc
 
 class ADO implements Serializable {
-
   static void triggerPipeline(def ctx, String namespace, String chartName, String chartVersion, Boolean hasDatabase) {
-    if (hasDatabase) {
-      triggerDatabasePipeline(ctx, chartName, chartVersion)
+    String service = getServiceName(namespace)
+    if(service) {
+      if (hasDatabase) {
+        triggerDatabasePipeline(ctx, service, chartName, chartVersion)
+      }
+      triggerHelmPipeline(ctx, service, namespace, chartName, chartVersion)
     }
-    triggerHelmPipeline(ctx, namespace, chartName, chartVersion)
   }
 
-  static void triggerDatabasePipeline(def ctx, String database, String version) {
+  static String getServiceName(String namespace) {
+    def services = [
+      'ffc-demo': 'demo',
+      'ffc-pay': 'payments',
+      'ffc-ahwr': 'vetvisits',
+      'ffc-pr': 'pr',
+      'ea-wq': 'ea-wq',
+      'ffc-grants': 'grants',
+      'ffc-mpdp': 'mpdp'
+    ]
+
+    return services[namespace]
```

Comment:
Correction: any language that uses backticks __OR__ the Elvis operator are ++ in my eyes

You took this use of Elvis away from me 😢 

BUT HERE IT IS ANYWAY

return services[namespace] ?: 'what happens when null'
return services[namespace]​ ?: {throw new RuntimeException('Unknown')}()

Kinda kewl, kinda just `??` in JS